--- bumblebee.conf	2016-08-21 08:03:14.443855488 +0300
+++ /etc/bumblebee/bumblebee.conf	2016-08-21 08:03:26.053855514 +0300
@@ -27,7 +27,7 @@
 [optirun]
 # Acceleration/ rendering bridge, possible values are auto, virtualgl and
 # primus.
-Bridge=auto
+Bridge=primus
 # The method used for VirtualGL to transport frames between X servers.
 # Possible values are proxy, jpeg, rgb, xv and yuv.
 VGLTransport=proxy
